src/testdir/test_breakindent.vim

```diff
@@ -759,6 +759,23 @@ func Test_breakindent20_list()
 	\ ]
   let lines = s:screen_lines2(1, 9, 20)
   call s:compare_lines(expect, lines)
+  " check formatlistpat indent
+  setl briopt+=list:-1
+  let &l:flp = '^\s*\d\+\.\?[\]:)}\t ]\s*'
+  redraw!
+  let expect = [
+	\ "  1.  Congress      ",
+	\ "      shall make no ",
+	\ "      law           ",
+	\ "  2.) Congress      ",
+	\ "      shall make no ",
+	\ "      law           ",
+	\ "  3.] Congress      ",
+	\ "      shall make no ",
+	\ "      law           ",
+	\ ]
+  let lines = s:screen_lines2(1, 9, 20)
+  call s:compare_lines(expect, lines)
   " reset linebreak option
   " Note: it indents by one additional
   " space, because of the leading space.
```

Comment:
please add another test with different levels of lists, so like this:
```
   * foobar with
     long text
   *** some other item
        with a lot of text
```

Reply:
also test with `briopt+=sbr:1`  or `briopt+=shift:2`

Reply:
It would be a bit more tricky -- I don't know how to start tests locally :)
Will add

Reply:
from the testdir, you can simply run `make test_breakindent.res` and then inspect messages